### PR TITLE
os_reload support in baremetal server

### PR DESCRIPTION
**Pull request by jianqiu**
Support os_reload rest api

### CODE REVIEW

**Comment by jianqiu** [comment]
@maximilien 

Could you please review this PR? Thanks!

File: lib/bluemix/softlayer/baremetal_utils.rb [hunk 1]
```diff
@@ -293,6 +293,30 @@ def self.create_baremetal(name, spec, place_order )
 
       end
 
+      def self.create_baremetal_2(name, spec, place_order )
```
Comment (maximilien) [inline lib/bluemix/softlayer/baremetal_utils.rb]: are you planning on removing `create_baremetal` if so then why not do it now?

**Comment by maximilien** [comment]
This seems like a duplicate of #5

**Comment by jianqiu** [comment]
Closing it since it is duplicated with [5](https://github.com/cloudfoundry-community/bosh-softlayer-baremetal-server/pull/5)

**Pull request closed without merging**